docs/cogs/information/user_info.md

```diff
@@ -1 +1,91 @@
 # User Info
+
+This cog contains some commands to show information about Discord user accounts.
+
+## `user`
+The `.user` command shows general information about a given user.
+
+```css
+.[userinfo|user|uinfo|userstats] [user]
+```
+
+|Argument|Required|Description|
+|:------:|:------:|:----------|
+|`user`||The user whose information is requested. If omitted, the requesting user receives their own information via direct messages.|
+
+!!! note
+    - Every user is allowed to request their own information.
+    - Requesting other users' information, however, requires the `user_info.view_userinfo` permission.
+
+The user information given by this command includes:
+
+- Current username, discriminator, avatar and id
+- Moderation statistics:
+    - Reports
+    - Warns
+    - Mutes
+    - Kicks
+    - Bans
+- Membership status
+- User activity
+- Active punishments
+
+!!! note
+    Moderation statistics:
+
+    - The term `active` refers to the number of times this user *has actively* reported/punished someone else.
+    - The term `passive` refers to the number of times this user *has been* reported/punished e.g. by a moderator.
+
+
+## `joined`
+The `.joined` command shows a rough estimate of how much time has passed since the last verification of a given member. If no `Verified` role has been configured yet, this estimate refers to the last join date of this member.
```

Comment:
Keep in mind that the actual calculation is probably going to be changed soon, so either remember to update this accordingly or make the documentation more generic here